docs/documentation/maya/simple_setup.md

```diff
@@ -1,221 +1,297 @@
 
-AdonisFX provides an endless array of customization and tweaking to fine tune the muscle dynamics by making use of these tools and the many additional parameters of the AdnMuscle deformer, that you can consult [here](muscle.md).
+To tweak additional parameters of the AdnMuscle deformer, the following documentation for [AdnMuscle](muscle.md) can be consulted.
 
-## AdnRibbonMuscle simple setup
+## AdnRibbonMuscle Simple Setup
 
-The process to set up an AdnRibbonMuscle is very similar to the one of setting up and AdnMuscle. It will essentially follow the same steps. We will start with the following elements:
+The process to setup an AdnRibbonMuscle is very similar to the one of setting up and AdnMuscle. It essentially follows the same steps. Start with the following elements:
 
  - An animated rig.
  - A simulated mesh.
 
-In this case we will simulate a planar muscle corresponding to a biceps, which will yield similar results to the case of the AdnMuscle deformer previously shown.
+In this case a planar muscle will be simulated corresponding to a biceps, which will yield similar results to the case of the AdnMuscle deformer previously shown.
 
 <figure>
   <img src="images/setup_ribbon_muscle_0.png"> 
-  <figcaption>Figure: Connection Editor tool.</figcaption>
+  <figcaption>Figure 16: Basic setup for planar biceps simulations.</figcaption>
 </figure>
 
 ### 1. Creating the deformer
 
-Similar to AdnMuscle, we will create the deformer by selecting the attachments (arm and forearm joints) and then the mesh we want to deform (the biceps muscle) and then pressing the ![adnMuscle](images/adn_ribbon_muscle.png) shelf button or go to *AdonisFX menu > Ribbon Muscle*.
+Similar to AdnMuscle, create the deformer by selecting the attachments (arm and forearm joints) and then the mesh to deform (the biceps muscle) and then pressing the ![AdnMuscle](images/adn_ribbon_muscle.png) shelf button or go to AdonisFX Menu > *Ribbon Muscle*.
 
 <figure>
   <img src="images/setup_ribbon_muscle_1.png"> 
-  <figcaption>Figure: Connection Editor tool.</figcaption>
+  <figcaption>Figure 17: AdnRibbonMuscle deformer creation scenario.</figcaption>
 </figure>
 
-If we want to create the AdnRibbonMuscle deformer with some initial customization we can double click the shelf button or press the option box in the menu item. This will display a popup window that will allow us to do some initial customization, as well as creating the deformer with a custom name. Once everything is in order press the "Create" button and the deformer will get created.
+To create the AdnRibbonMuscle deformer with some initial customization, double click the shelf button or press the option box in the menu item. This will display a pop-up window that will allow to do some initial customization, as well as creating the deformer with a custom name. Once all data has been provided press the "Create" button and the deformer will get created.
```

Comment:
```suggestion
To create the AdnRibbonMuscle deformer with some initial customization, double-click the shelf button or press the option box in the menu item. This will display a pop-up window that will allow to do some initial customization, as well as creating the deformer with a custom name. Once all data has been provided press the *Create* button and the deformer will get created.
```

Reply:
Done.